src/Compilers/CSharp/Test/Syntax/Parsing/LocalFunctionParsingTests.cs

```diff
@@ -648,6 +648,91 @@ void M()
                 Diagnostic(ErrorCode.ERR_EOFExpected, "}").WithLocation(8, 1));
         }
 
+        [Fact]
+        [WorkItem(38801, "https://github.com/dotnet/roslyn/issues/38801")]
+        public void LocalFunctionAttribute_Error_LocalVariable_MultipleDeclarators()
+        {
+            var tree = UsingTree(@"
+class C
+{
+    void M()
+    {
+        [A] object local1, local2;
```

Comment:
What hapens if the attribute is before the second local?

Reply:
When the attribute comes after the type the parser believes it's some kind of bad array declarator.